src/App.tsx

```diff
@@ -1,14 +1,74 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { ShowType } from './types/ShowType';
 
 export const App: React.FC = () => {
+  const [todosFromServer, setTodosFromServer] = useState<Todo[]>([]);
+  const [show, setShow] = useState<ShowType>(ShowType.all);
+  const [filter, setFilter] = useState('');
```

Comment:
From the the first glance it is hard to understand the purpose of these states, consider renaming them to be more descriptive

Reply:
changed to todosShowMode and todosFilterByTitle... Hope it's better